### PR TITLE
cmd: add output options for cilium monitor

### PR DESCRIPTION
**Pull request by ianvernon**
By default, when `cilium monitor` is ran, a one-line summary will be output, only for drop and capture messages.
A new flag, -v for verbose, is added which will output more detailed information about each packet for debug, drop, and capture messages, which was before the default behavior for `cilium monitor`.
    
Signed-off by: Ian Vernon <ian@covalent.io>

### CODE REVIEW

**Comment by ianvernon** [comment]
@tgraf this is a very basic version of the changes proposed in #901. Please let me know if this is on the right track.

**Comment by tgraf** [comment]
> @tgraf this is a very basic version of the changes proposed in #901. Please let me know if this is on the right track.

Yes! This is exactly on the right track. I think we can find a better wording. `CAPTURE` may not be clear to the user. Let's merge something minimal for now to ease the troubleshooting process.

**Comment by ianvernon** [comment]
@tgraf how about "SENDING" or "FORWARDING"?

**Comment by tgraf** [comment]
> @tgraf how about "SENDING" or "FORWARDING"?

We have 4 main actions:
 - the packet enters cilium processing `in`
 - the packet is being processed `process` (we can possibly skip these in non-verbose)
 - the packet is forwarded `out`
 - the packet is dropped `drop`

Maybe something like this?

`in endpoint 200 > 10.1.1.1`
`out to-proxy`
`in netdev > 10.1.1.1`
`out endpoint 300`

`in netdev > 20.1.1.1`
`drop (policy denied)`

`in overlay > 30.1.1.1`
`out overlay (1.1.1.1)`

**Comment by ianvernon** [comment]
OK, I went through the DebugMessage types and pulled out the ones I thought were relevant to what you mentioned above, and reformatted the others accordingly. Currently not all of what you want above is achievable as part of this first pass at this issue (i.e., getting the IPs from the various message types) - it will be once I add more rich metadata as we discussed by querying the API. Let's discuss further on Slack if this isn't what you are looking for so I can nail down the requirements. 

This is what the monitor looks like when ran against the `11-getting-started.sh` test:
```
vagrant@cilium-master:~/go/src/github.com/cilium/cilium$ sudo cilium monitor 
Listening for events on 2 CPUs with 64x4096 of shared memory
Press Ctrl-C to quit
from [container 391 / endpoint 29898]
drop [ifindex 391 / endpoint 29898] (Invalid destination mac)
from [container 391 / endpoint 29898]
drop [ifindex 391 / endpoint 29898] (Invalid destination mac)
from [container 391 / endpoint 29898]
drop [ifindex 391 / endpoint 29898] (Unknown L3 target address)
from [container 391 / endpoint 29898]
drop [ifindex 391 / endpoint 29898] (Invalid source mac)
from [container 391 / endpoint 29898]
drop [ifindex 391 / endpoint 29898] (Invalid destination mac)
from [container 391 / endpoint 29898]
drop [ifindex 391 / endpoint 29898] (Invalid destination mac)
from [container 391 / endpoint 29898]
drop [ifindex 391 / endpoint 29898] (Invalid source mac)
from [container 391 / endpoint 29898]
drop [ifindex 391 / endpoint 29898] (Invalid destination mac)
from [container 391 / endpoint 29898]
drop [ifindex 391 / endpoint 29898] (Invalid source mac)
from [container 393 / endpoint 33115]
drop [ifindex 393 / endpoint 33115] (Policy denied)
from [container 393 / endpoint 33115]
from [endpoint 33115] > [ifindex 393]
from [container 393 / endpoint 33115]
drop [ifindex 393 / endpoint 33115] (Policy denied)
from [container 393 / endpoint 33115]
drop [ifindex 393 / endpoint 33115] (Policy denied)
from [container 393 / endpoint 33115]
drop [ifindex 393 / endpoint 33115] (Policy denied)
from [container 393 / endpoint 33115]
drop [ifindex 393 / endpoint 33115] (Policy denied)
from [container 393 / endpoint 33115]
drop [ifindex 393 / endpoint 33115] (Policy denied)
from [container 393 / endpoint 33115]
drop [ifindex 393 / endpoint 33115] (Policy denied)
from [container 393 / endpoint 33115]
drop [ifindex 393 / endpoint 33115] (Policy denied)
from [container 393 / endpoint 33115]
drop [ifindex 393 / endpoint 33115] (Policy denied)
from [container 393 / endpoint 33115]
drop [ifindex 393 / endpoint 33115] (Policy denied)
from [container 393 / endpoint 33115]
drop [ifindex 393 / endpoint 33115] (Policy denied)
from [container 393 / endpoint 33115]
drop [ifindex 393 / endpoint 33115] (Policy denied)
from [container 391 / endpoint 29898]
drop [ifindex 391 / endpoint 29898] (Invalid destination mac)
from [container 393 / endpoint 33115]
from [seclabel 420] > [container 29898]
drop [ifindex 391 / endpoint 29898] (Policy denied)
from [container 393 / endpoint 33115]
from [seclabel 420] > [container 29898]
drop [ifindex 391 / endpoint 29898] (Policy denied)
from [container 393 / endpoint 33115]
drop [ifindex 393 / endpoint 33115] (Invalid destination mac)
from [container 391 / endpoint 29898]
drop [ifindex 391 / endpoint 29898] (Invalid destination mac)
from [container 393 / endpoint 33115]
drop [ifindex 393 / endpoint 33115] (Invalid destination mac)
from [container 395 / endpoint 64189]
drop [ifindex 395 / endpoint 64189] (Policy denied)
from [container 395 / endpoint 64189]
from [endpoint 64189] > [ifindex 395]
from [container 395 / endpoint 64189]
drop [ifindex 395 / endpoint 64189] (Policy denied)
from [container 395 / endpoint 64189]
drop [ifindex 395 / endpoint 64189] (Policy denied)
from [container 395 / endpoint 64189]
drop [ifindex 395 / endpoint 64189] (Policy denied)
from [container 395 / endpoint 64189]
drop [ifindex 395 / endpoint 64189] (Policy denied)
from [container 395 / endpoint 64189]
drop [ifindex 395 / endpoint 64189] (Policy denied)
from [container 395 / endpoint 64189]
drop [ifindex 395 / endpoint 64189] (Policy denied)
from [container 395 / endpoint 64189]
drop [ifindex 395 / endpoint 64189] (Policy denied)
from [container 395 / endpoint 64189]
drop [ifindex 395 / endpoint 64189] (Policy denied)
from [container 395 / endpoint 64189]
drop [ifindex 395 / endpoint 64189] (Policy denied)
from [container 395 / endpoint 64189]
drop [ifindex 395 / endpoint 64189] (Policy denied)
from [container 395 / endpoint 64189]
drop [ifindex 395 / endpoint 64189] (Policy denied)
from [container 395 / endpoint 64189]
from [seclabel 421] > [container 29898]
from [endpoint 29898] > [ifindex 391]
from [container 391 / endpoint 29898]
from [endpoint 29898] > [ifindex 391]
from [container 391 / endpoint 29898]
from [seclabel 419] > [container 64189]
from [endpoint 64189] > [ifindex 395]
from [container 395 / endpoint 64189]
from [seclabel 421] > [container 29898]
from [endpoint 29898] > [ifindex 391]
from [container 391 / endpoint 29898]
from [seclabel 419] > [container 64189]
from [endpoint 64189] > [ifindex 395]
from [container 395 / endpoint 64189]
drop [ifindex 395 / endpoint 64189] (Invalid destination mac)
from [container 397 / endpoint 38061]
drop [ifindex 397 / endpoint 38061] (Policy denied)
from [container 397 / endpoint 38061]
from [endpoint 38061] > [ifindex 397]
from [container 397 / endpoint 38061]
drop [ifindex 397 / endpoint 38061] (Policy denied)
from [container 397 / endpoint 38061]
drop [ifindex 397 / endpoint 38061] (Policy denied)
from [container 397 / endpoint 38061]
drop [ifindex 397 / endpoint 38061] (Policy denied)
from [container 397 / endpoint 38061]
drop [ifindex 397 / endpoint 38061] (Policy denied)
from [container 397 / endpoint 38061]
drop [ifindex 397 / endpoint 38061] (Policy denied)
from [container 397 / endpoint 38061]
drop [ifindex 397 / endpoint 38061] (Policy denied)
from [container 397 / endpoint 38061]
drop [ifindex 397 / endpoint 38061] (Policy denied)
from [container 397 / endpoint 38061]
drop [ifindex 397 / endpoint 38061] (Policy denied)
from [container 397 / endpoint 38061]
drop [ifindex 397 / endpoint 38061] (Policy denied)
from [container 397 / endpoint 38061]
drop [ifindex 397 / endpoint 38061] (Policy denied)
from [container 397 / endpoint 38061]
drop [ifindex 397 / endpoint 38061] (Policy denied)
from [container 397 / endpoint 38061]
drop [ifindex 397 / endpoint 38061] (Policy denied)
from [container 397 / endpoint 38061]
drop [ifindex 397 / endpoint 38061] (Policy denied)
from [container 397 / endpoint 38061]
drop [ifindex 397 / endpoint 38061] (Policy denied)
from [container 391 / endpoint 29898]
drop [ifindex 391 / endpoint 29898] (Invalid destination mac)
from [container 397 / endpoint 38061]
drop [ifindex 397 / endpoint 38061] (Invalid destination mac)
from [container 397 / endpoint 38061]
from [seclabel 422] > [container 29898]
from [endpoint 29898] > [ifindex 391]
from [container 391 / endpoint 29898]
from [seclabel 419] > [container 38061]
from [endpoint 38061] > [ifindex 397]
from [container 397 / endpoint 38061]
from [seclabel 422] > [container 29898]
from [endpoint 29898] > [ifindex 391]
from [container 397 / endpoint 38061]
from [seclabel 422] > [container 29898]
from [endpoint 29898] > [ifindex 391]
from [container 391 / endpoint 29898]
from [seclabel 419] > [container 38061]
from [endpoint 38061] > [ifindex 397]
from [container 391 / endpoint 29898]
from [seclabel 419] > [container 38061]
from [endpoint 38061] > [ifindex 397]
from [container 397 / endpoint 38061]
from [seclabel 422] > [container 29898]
from [endpoint 29898] > [ifindex 391]
from [container 397 / endpoint 38061]
from [seclabel 422] > [container 29898]
from [endpoint 29898] > [ifindex 391]
from [container 391 / endpoint 29898]
from [seclabel 419] > [container 38061]
from [endpoint 38061] > [ifindex 397]
from [container 397 / endpoint 38061]
drop [ifindex 397 / endpoint 38061] (CT: Can't create entry from packet)
from [container 391 / endpoint 29898]
drop [ifindex 391 / endpoint 29898] (CT: Can't create entry from packet)
from [container 391 / endpoint 29898]
drop [ifindex 391 / endpoint 29898] (CT: Can't create entry from packet)
from [container 391 / endpoint 29898]
drop [ifindex 391 / endpoint 29898] (CT: Can't create entry from packet)
from [container 399 / endpoint 11004]
drop [ifindex 399 / endpoint 11004] (Policy denied)
from [container 399 / endpoint 11004]
from [endpoint 11004] > [ifindex 399]
from [container 399 / endpoint 11004]
drop [ifindex 399 / endpoint 11004] (Policy denied)
from [container 391 / endpoint 29898]
drop [ifindex 391 / endpoint 29898] (CT: Can't create entry from packet)
from [container 399 / endpoint 11004]
drop [ifindex 399 / endpoint 11004] (Policy denied)
from [container 399 / endpoint 11004]
drop [ifindex 399 / endpoint 11004] (Policy denied)
from [container 399 / endpoint 11004]
drop [ifindex 399 / endpoint 11004] (Policy denied)
from [container 399 / endpoint 11004]
drop [ifindex 399 / endpoint 11004] (Policy denied)
from [container 399 / endpoint 11004]
drop [ifindex 399 / endpoint 11004] (Policy denied)
from [container 399 / endpoint 11004]
drop [ifindex 399 / endpoint 11004] (Policy denied)
from [container 399 / endpoint 11004]
drop [ifindex 399 / endpoint 11004] (Policy denied)
from [container 399 / endpoint 11004]
drop [ifindex 399 / endpoint 11004] (Policy denied)
from [container 391 / endpoint 29898]
drop [ifindex 391 / endpoint 29898] (CT: Can't create entry from packet)
from [container 399 / endpoint 11004]
drop [ifindex 399 / endpoint 11004] (Policy denied)
from [container 399 / endpoint 11004]
drop [ifindex 399 / endpoint 11004] (Policy denied)
from [container 399 / endpoint 11004]
drop [ifindex 399 / endpoint 11004] (Policy denied)
from [container 399 / endpoint 11004]
drop [ifindex 399 / endpoint 11004] (Policy denied)
from [container 399 / endpoint 11004]
drop [ifindex 399 / endpoint 11004] (Policy denied)
from [container 391 / endpoint 29898]
drop [ifindex 391 / endpoint 29898] (CT: Can't create entry from packet)
from [container 399 / endpoint 11004]
drop [ifindex 399 / endpoint 11004] (Invalid destination mac)
from [container 399 / endpoint 11004]
from [seclabel 423] > [container 29898]
from [endpoint 29898] > [ifindex 391]
from [container 391 / endpoint 29898]
from [seclabel 419] > [container 11004]
from [endpoint 11004] > [ifindex 399]
from [container 399 / endpoint 11004]
from [seclabel 423] > [container 29898]
from [endpoint 29898] > [ifindex 391]
from [container 399 / endpoint 11004]
from [seclabel 423] > [container 29898]
from [endpoint 29898] > [ifindex 391]
from [container 391 / endpoint 29898]
from [seclabel 419] > [container 11004]
from [endpoint 11004] > [ifindex 399]
from [container 391 / endpoint 29898]
from [seclabel 419] > [container 11004]
from [endpoint 11004] > [ifindex 399]
from [container 399 / endpoint 11004]
from [seclabel 423] > [container 29898]
from [endpoint 29898] > [ifindex 391]
from [container 399 / endpoint 11004]
from [seclabel 423] > [container 29898]
from [endpoint 29898] > [ifindex 391]
from [container 391 / endpoint 29898]
from [seclabel 419] > [container 11004]
from [endpoint 11004] > [ifindex 399]
from [container 399 / endpoint 11004]
drop [ifindex 399 / endpoint 11004] (CT: Can't create entry from packet)
from [container 391 / endpoint 29898]
drop [ifindex 391 / endpoint 29898] (CT: Can't create entry from packet)
from [container 391 / endpoint 29898]
drop [ifindex 391 / endpoint 29898] (CT: Can't create entry from packet)
from [container 391 / endpoint 29898]
drop [ifindex 391 / endpoint 29898] (CT: Can't create entry from packet)
from [container 391 / endpoint 29898]
drop [ifindex 391 / endpoint 29898] (CT: Can't create entry from packet)
from [container 391 / endpoint 29898]
drop [ifindex 391 / endpoint 29898] (CT: Can't create entry from packet)
from [container 401 / endpoint 49508]
drop [ifindex 401 / endpoint 49508] (Policy denied)
from [container 401 / endpoint 49508]
from [endpoint 49508] > [ifindex 401]
from [container 401 / endpoint 49508]
drop [ifindex 401 / endpoint 49508] (Policy denied)
from [container 401 / endpoint 49508]
drop [ifindex 401 / endpoint 49508] (Policy denied)
from [container 391 / endpoint 29898]
drop [ifindex 391 / endpoint 29898] (CT: Can't create entry from packet)
from [container 401 / endpoint 49508]
drop [ifindex 401 / endpoint 49508] (Policy denied)
from [container 401 / endpoint 49508]
drop [ifindex 401 / endpoint 49508] (Policy denied)
from [container 401 / endpoint 49508]
drop [ifindex 401 / endpoint 49508] (Policy denied)
from [container 401 / endpoint 49508]
drop [ifindex 401 / endpoint 49508] (Policy denied)
from [container 401 / endpoint 49508]
drop [ifindex 401 / endpoint 49508] (Policy denied)
from [container 401 / endpoint 49508]
drop [ifindex 401 / endpoint 49508] (Policy denied)
from [container 401 / endpoint 49508]
drop [ifindex 401 / endpoint 49508] (Policy denied)
from [container 401 / endpoint 49508]
drop [ifindex 401 / endpoint 49508] (Invalid source mac)
from [container 401 / endpoint 49508]
from [endpoint 49508] > [ifindex 401]
from [container 401 / endpoint 49508]
to host, policy-skip=0
from [endpoint 49508] > [ifindex 388]
from [netdev 388 / endpoint 0]
from [seclabel 1] > [container 49508]
from [endpoint 49508] > [ifindex 401]
from [container 401 / endpoint 49508]
to host, policy-skip=0
from [endpoint 49508] > [ifindex 388]
from [container 401 / endpoint 49508]
to host, policy-skip=0
from [endpoint 49508] > [ifindex 388]
from [netdev 388 / endpoint 0]
from [seclabel 1] > [container 49508]
from [endpoint 49508] > [ifindex 401]
from [container 391 / endpoint 29898]
drop [ifindex 391 / endpoint 29898] (CT: Can't create entry from packet)
from [container 401 / endpoint 49508]
drop [ifindex 401 / endpoint 49508] (Invalid source mac)
from [container 401 / endpoint 49508]
drop [ifindex 401 / endpoint 49508] (Invalid source mac)
from [netdev 388 / endpoint 0]
from [seclabel 1] > [container 29898]
from [endpoint 29898] > [ifindex 391]
from [container 391 / endpoint 29898]
from [endpoint 29898] > [ifindex 391]
from [container 391 / endpoint 29898]
to stack, policy-skip=1
from [endpoint 0] > [ifindex 0]
from [netdev 388 / endpoint 0]
from [seclabel 1] > [container 29898]
from [endpoint 29898] > [ifindex 391]
from [container 391 / endpoint 29898]
from [endpoint 29898] > [ifindex 391]
from [netdev 388 / endpoint 0]
from [seclabel 1] > [container 29898]
from [endpoint 29898] > [ifindex 391]
from [container 391 / endpoint 29898]
to stack, policy-skip=1
from [endpoint 0] > [ifindex 0]
from [container 391 / endpoint 29898]
to stack, policy-skip=1
from [endpoint 0] > [ifindex 0]
from [netdev 388 / endpoint 0]
from [seclabel 1] > [container 29898]
from [endpoint 29898] > [ifindex 391]
from [netdev 388 / endpoint 0]
from [seclabel 1] > [container 49508]
from [endpoint 49508] > [ifindex 401]
from [container 401 / endpoint 49508]
to host, policy-skip=0
from [endpoint 49508] > [ifindex 388]
from [container 401 / endpoint 49508]
to host, policy-skip=0
from [endpoint 49508] > [ifindex 388]
from [netdev 388 / endpoint 0]
from [seclabel 1] > [container 49508]
from [endpoint 49508] > [ifindex 401]
from [container 401 / endpoint 49508]
drop [ifindex 401 / endpoint 49508] (CT: Can't create entry from packet)
from [netdev 388 / endpoint 0]
from [seclabel 1] > [container 49508]
drop [ifindex 388 / endpoint 49508] (CT: Can't create entry from packet)
from [netdev 388 / endpoint 0]
from [seclabel 1] > [container 49508]
drop [ifindex 388 / endpoint 49508] (CT: Can't create entry from packet)
from [netdev 388 / endpoint 0]
from [seclabel 1] > [container 49508]
drop [ifindex 388 / endpoint 49508] (CT: Can't create entry from packet)
from [netdev 388 / endpoint 0]
from [seclabel 1] > [container 49508]
drop [ifindex 388 / endpoint 49508] (CT: Can't create entry from packet)
from [container 401 / endpoint 49508]
drop [ifindex 401 / endpoint 49508] (Invalid destination mac)
from [netdev 388 / endpoint 0]
from [container 403 / endpoint 3365]
drop [ifindex 403 / endpoint 3365] (Policy denied)
from [container 403 / endpoint 3365]
from [endpoint 3365] > [ifindex 403]
from [container 403 / endpoint 3365]
drop [ifindex 403 / endpoint 3365] (Policy denied)
from [container 403 / endpoint 3365]
drop [ifindex 403 / endpoint 3365] (Policy denied)
from [container 391 / endpoint 29898]
to stack, policy-skip=1
from [endpoint 0] > [ifindex 0]
from [netdev 388 / endpoint 0]
from [seclabel 1] > [container 29898]
from [endpoint 29898] > [ifindex 391]
from [container 391 / endpoint 29898]
drop [ifindex 391 / endpoint 29898] (CT: Can't create entry from packet)
from [container 403 / endpoint 3365]
drop [ifindex 403 / endpoint 3365] (Policy denied)
from [container 403 / endpoint 3365]
drop [ifindex 403 / endpoint 3365] (Policy denied)
from [netdev 388 / endpoint 0]
from [seclabel 1] > [container 29898]
drop [ifindex 388 / endpoint 29898] (CT: Can't create entry from packet)
from [netdev 388 / endpoint 0]
from [seclabel 1] > [container 29898]
drop [ifindex 388 / endpoint 29898] (CT: Can't create entry from packet)
from [container 403 / endpoint 3365]
drop [ifindex 403 / endpoint 3365] (Policy denied)
from [container 403 / endpoint 3365]
drop [ifindex 403 / endpoint 3365] (Policy denied)
from [netdev 388 / endpoint 0]
from [seclabel 1] > [container 29898]
drop [ifindex 388 / endpoint 29898] (CT: Can't create entry from packet)
from [container 391 / endpoint 29898]
drop [ifindex 391 / endpoint 29898] (CT: Can't create entry from packet)
from [container 403 / endpoint 3365]
drop [ifindex 403 / endpoint 3365] (Policy denied)
from [container 403 / endpoint 3365]
drop [ifindex 403 / endpoint 3365] (Policy denied)
from [container 403 / endpoint 3365]
drop [ifindex 403 / endpoint 3365] (Policy denied)
from [netdev 388 / endpoint 0]
from [seclabel 1] > [container 29898]
drop [ifindex 388 / endpoint 29898] (CT: Can't create entry from packet)
from [container 403 / endpoint 3365]
drop [ifindex 403 / endpoint 3365] (Policy denied)
from [container 403 / endpoint 3365]
drop [ifindex 403 / endpoint 3365] (Policy denied)
from [netdev 388 / endpoint 0]
from [container 403 / endpoint 3365]
drop [ifindex 403 / endpoint 3365] (Policy denied)
from [container 403 / endpoint 3365]
drop [ifindex 403 / endpoint 3365] (Policy denied)
from [container 403 / endpoint 3365]
pre-proxy port 20480 [endpoint 3365]
post-proxy port 4135 [endpoint 3365]
to host, policy-skip=0
from [endpoint 3365] > [ifindex 388]
from [netdev 388 / endpoint 0]
pre-proxy port 0 [endpoint 0]
post-proxy port 0 [endpoint 0]
from [seclabel 1] > [container 3365]
from [endpoint 3365] > [ifindex 403]
from [container 403 / endpoint 3365]
pre-proxy port 20480 [endpoint 3365]
post-proxy port 4135 [endpoint 3365]
to host, policy-skip=0
from [endpoint 3365] > [ifindex 388]
from [container 403 / endpoint 3365]
pre-proxy port 20480 [endpoint 3365]
post-proxy port 4135 [endpoint 3365]
to host, policy-skip=0
from [endpoint 3365] > [ifindex 388]
from [netdev 388 / endpoint 0]
pre-proxy port 0 [endpoint 0]
post-proxy port 0 [endpoint 0]
from [seclabel 1] > [container 3365]
from [endpoint 3365] > [ifindex 403]
from [netdev 388 / endpoint 0]
from [seclabel 1] > [container 29898]
drop [ifindex 388 / endpoint 29898] (CT: Can't create entry from packet)
from [netdev 388 / endpoint 0]
pre-proxy port 0 [endpoint 0]
post-proxy port 0 [endpoint 0]
from [seclabel 1] > [container 3365]
from [endpoint 3365] > [ifindex 403]
from [container 403 / endpoint 3365]
pre-proxy port 20480 [endpoint 3365]
post-proxy port 4135 [endpoint 3365]
to host, policy-skip=0
from [endpoint 3365] > [ifindex 388]
from [container 403 / endpoint 3365]
pre-proxy port 20480 [endpoint 3365]
post-proxy port 4135 [endpoint 3365]
to host, policy-skip=0
from [endpoint 3365] > [ifindex 388]
from [netdev 388 / endpoint 0]
pre-proxy port 0 [endpoint 0]
post-proxy port 0 [endpoint 0]
from [seclabel 1] > [container 3365]
from [endpoint 3365] > [ifindex 403]
from [container 403 / endpoint 3365]
drop [ifindex 403 / endpoint 3365] (CT: Can't create entry from packet)
from [netdev 388 / endpoint 0]
pre-proxy port 0 [endpoint 0]
post-proxy port 0 [endpoint 0]
from [netdev 388 / endpoint 0]
pre-proxy port 0 [endpoint 0]
post-proxy port 0 [endpoint 0]
from [container 391 / endpoint 29898]
drop [ifindex 391 / endpoint 29898] (CT: Can't create entry from packet)
from [netdev 388 / endpoint 0]
pre-proxy port 0 [endpoint 0]
post-proxy port 0 [endpoint 0]
from [netdev 388 / endpoint 0]
from [seclabel 1] > [container 29898]
drop [ifindex 388 / endpoint 29898] (CT: Can't create entry from packet)
```

**Review by tgraf — APPROVED** [review]

**Comment by tgraf** [comment]
Looks good but will need a rebase. It will conflict due to my BPF changes so `daemon/bpf.sha` will have to be regenerated for each commit which modifies `bpf/`

File: pkg/bpfdebug/drop.go [hunk 1]
```diff
@@ -81,8 +81,15 @@ func dropReason(reason uint8) string {
 	return fmt.Sprintf("%d", reason)
 }
 
+// DumpInfo https://techcrunch.com/2017/07/12/soundshroud/
+func (n *DropNotify) DumpInfo(data []byte) {
+	fmt.Printf("xx drop (%s) to endpoint %d, identity %d->%d: %s\n",
+		dropReason(n.SubType), n.DstID, n.SrcLabel, n.DstLabel,
```
Comment (aalemayhu) [inline pkg/bpfdebug/drop.go:87]: Why is this prefixed `xx`?

**Review by aalemayhu — COMMENTED** [review]
Overall this is an improvement but looking at the output this seems rushed IMO. For example the filter output now looks weird (`sudo ./cilium/cilium monitor --type debug -v`).

File: pkg/bpfdebug/drop.go (hunk 1 above)
Comment (tgraf) [reply pkg/bpfdebug/drop.go:87]: It indicates a dropped packet

**Pull request merged**